### ISSUE TITLE
Audit Multiple Databases

### ISSUE THREAD

**Issue by ben007zxbaj** [owen-it/laravel-auditing#649]
How to audit tables on different database server im using mongodb for the audit table. and multiple mysql server for the database

**Comment by Tofandel** [comment]
Probably a dupe of #508 but it's hard to say without more information on how you access those databases

**Comment by erikn69** [comment]
Change the config on runtime (on creating, updating, deleting)
`config(['audit.drivers.database.table' => 'other_database.audit']);`

**Comment by MortenDHansen** [comment]
Hi. It shouldn't be a problem having multiple db connections. 
Could you please help replicate the issue? 

Describe what you are doing, and what the problem is so that i can create a test-replica of the problem. 😄

**Comment by ben007zxbaj** [comment]
i have two different mysql servers and my audits is save via mongodb


but only the main mysql server is being audited

regards

**Comment by MortenDHansen** [comment]
Have you defined the connection in the model?

**Comment by ben007zxbaj** [comment]
yes but that's last year i will try it again will update you the status if
its working

thanks 👍
